nucypher/blockchain/eth/chains.py

```diff
@@ -1,88 +1,74 @@
-import random
-from abc import ABC
-from typing import List
+from typing import List, Union
 
-from nucypher.blockchain.eth.constants import NucypherMinerConfig
-from nucypher.blockchain.eth.interfaces import ContractProvider
+from nucypher.blockchain.eth.constants import NucypherMinerConstants
+from nucypher.blockchain.eth.interfaces import ContractInterface, DeployerInterface
 
 
-class TheBlockchain(ABC):
-    """
-    http://populus.readthedocs.io/en/latest/config.html#chains
-
-    mainnet: Connects to the public ethereum mainnet via geth.
-    ropsten: Connects to the public ethereum ropsten testnet via geth.
-    tester: Uses an ephemeral in-memory chain backed by pyethereum.
-    testrpc: Uses an ephemeral in-memory chain backed by pyethereum.
-    temp: Local private chain whos data directory is removed when the chain is shutdown. Runs via geth.
-    """
+class Blockchain:
+    """A view of a blockchain through a provided interface"""
```

Comment:
Do we have docs/docstrings on the previous types of testnets we can connect to and how to connect to them? Since this was removed, I want to make sure it's really obvious about _which_ network we're connecting to, especially by default.

Reply:
Yes - this may need a bigger docstring;  `network_name` was moved over to `BlockchainConfig`.